marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/writer/WriteDataWithOptionsTest.java

```diff
@@ -6,13 +6,16 @@
 import com.marklogic.hub.spark.sql.sources.v2.AbstractSparkConnectorTest;
 import org.junit.jupiter.api.Test;
 
+import org.apache.spark.sql.sources.v2.writer.WriterCommitMessage;
```

Comment:
Just do a revert on this file so that there are no changes to it

Reply:
ready for commit